frontend/src/containers/SearchTable.jsx

```diff
@@ -85,7 +85,7 @@ export default class SearchTable extends Component {
             isLoading: true
         }, async () => {
             const url = this.state.geneSearch ?
-                `http://localhost:8000/variants/?page=${this.state.currentPage}&geneSearch=${this.state.geneSearch}` :
+                `http://localhost:8000/variants/?page=${this.state.currentPage}&search=${this.state.geneSearch}` :
```

Comment:
not sure if this one was intentional?

Reply:
Yes, "search" is the parameter that the SearchFilter uses. One of the changes I made was to replace the search implementation in the endpoint with SearchFilter from the DRF.